### DISCUSSION TITLE
Proposing a breaking change to remove mapbox-gl-supported

### DISCUSSION THREAD

**Discussion by zhangyiatmicrosoft** [maplibre/maplibre-gl-js#2452] [Ideas]
We are only using an object literal, and then just export it.
https://github.com/mapbox/mapbox-gl-supported

This library has not had any meaningful update for 3 years and is becoming stale.
Anyone out there who still needs it can easily reference or add to their repo instead of using MapLibre as the middleman.

Draft PR here:
https://github.com/maplibre/maplibre-gl-js/pull/2451

If everyone agrees I will write changelog and finish it. Net saving on minimized script is 2161 bytes, which is a good return for several lines of changes.

**Comment by HarelM** [comment]
I generally think this is a good direction, see my comments in the PR.

**Reply by zhangyiatmicrosoft** [reply]
Thank you very much!
I will push the PR through
